Mathlib/SetTheory/Cardinal/HasCardinalLT.lean

```diff
@@ -0,0 +1,78 @@
+/-
+Copyright (c) 2024 Joël Riou. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Joël Riou
+-/
+import Mathlib.SetTheory.Cardinal.Basic
+
+/-!
+# The property of being of cardinality less than a cardinal
+
+Given `X : Type u` and `κ : Cardinal.{v}`, we introduce a predicate
+`HasCardinalLT X κ` expressing that
+`Cardinal.mk (ULift.{v} X) < Cardinal.lift κ`.
+
+-/
+
+universe w v u u'
+
+/-- The property that the cardinal of a type `X : Type u` is less than `κ : Cardinal.{v}`. -/
+def HasCardinalLT (X : Type u) (κ : Cardinal.{v}) : Prop :=
+  Cardinal.mk (ULift.{v} X) < Cardinal.lift κ
+
+lemma hasCardinalLT_iff_cardinal_mk_lt (X : Type u) (κ : Cardinal.{u}) :
+    HasCardinalLT X κ ↔ Cardinal.mk X < κ := by
+  simp [HasCardinalLT]
+
+namespace HasCardinalLT
+
+section
+
+variable {X : Type u} {κ : Cardinal.{v}} (h : HasCardinalLT X κ)
+
+include h
+
+lemma small : Small.{v} X := by
```

Comment:
I do like this lemma, and I suspect it could be proven a bit more easily using [`Cardinal.small_iff_lift_mk_lt_univ`](https://leanprover-community.github.io/mathlib4_docs/Mathlib/SetTheory/Ordinal/Basic.html#Cardinal.small_iff_lift_mk_lt_univ).

Reply:
Thanks!